operator/config/crd/bases/forklift.konveyor.io_plans.yaml

```
@@ -222,6 +222,9 @@ spec:
                 - destination
                 - source
                 type: object
+              selectedInstanceType:
```

```suggestion
              instanceType:
```
(it's obvious that it's selected, like the target namespace that is selected for the plan)